pkgs/applications/graphics/guetzli/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, libpng, fetchgit, pkgconfig }:
+stdenv.mkDerivation {
+  name = "guetzli-1.0.1";
+  src = fetchgit {
```

Comment:
Please use `fetchFromGitHub` like so, if it is possible to get rid of `fetchSubmodules = true`:
`src = fetchFromGitHub {
  owner = "google";
  repo = "guetzli";
  rev = "v${version}"
  sha256 = "…";
}`

To be able to do that, you have to declare `version` separately and make the attribute set recursive.